libs/ui/src/lib/facet/checkbox-input/checkbox-input.component.ts

```diff
@@ -0,0 +1,33 @@
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core'
```

Comment:
I think the name is not explicit enough, there the checkbox, but also the invert. And `CheckboxInput` is confusing as a name.
I prefer something like `FacetItem`
We can agree on those terms:
- facet list (all facets on the left)
- facet block: a block for a particular definition (usually an index field)
- facet item (one line in a block)
What do you think ?

Reply:
It will be more precise indeed. I'll also rename the facet module to facets to match with the search lib.